.gitignore

```diff
@@ -1,5 +1,2 @@
 phpunit.xml
-vendor
-composer.lock
```

Comment:
I talked with @naderman about this at FOSDEM. It's ignored for projects that depend on ODM, so it doesn't affect our users, but it gives developers consistency for reproducing test results (also the "build status" icon).